packages/plugins/rubygem-foreman_templates/rubygem-foreman_templates.spec

```diff
@@ -1,49 +1,36 @@
 # template: foreman_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
 %global gem_name foreman_templates
 %global plugin_name templates
-%global foreman_min_version 1.24.0
-
-Summary:    Template-syncing engine for Foreman
-Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    9.3.0
-Release:    2%{?foremandist}%{?dist}
-Group:      Applications/Systems
-License:    GPLv3
-URL:        https://github.com/theforeman/foreman_templates
-Source0:    https://rubygems.org/gems/%{gem_name}-%{version}.gem
-
-Requires: git
-BuildRequires: git
+%global foreman_min_version 3.7
+
+Name: rubygem-%{gem_name}
+Version: 9.4.0
+Release: 1%{?foremandist}%{?dist}
+Summary: Template-syncing engine for Foreman
+License: GPLv3
+URL: https://github.com/theforeman/foreman_templates
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
 # start specfile generated dependencies
```

Comment:
It's better to explicitly require 1.18.0+ since that contains the dependency on `git-core`:
```suggestion
# Duplicates the autogenerated one so we can ensure git-core is pulled in
Requires: rubygem(git) >= 1.18.0

# start specfile generated dependencies
```

Reply:
Right, updated.